DigitalOffice/Tests/ProjectServiceTests/CommandsTests/CreateNewProjectCommandTests.cs

```diff
@@ -66,7 +66,7 @@ public void FailCreateNewProjectMatchOfProjectNameTests()
                  .Returns(true);
 
             mapperMock
-                .Setup(X => X.Map(It.IsAny<NewProjectRequest>()))
```

Comment:
Once we have a branch to fix bugs in the project service, we can copy these changes there. But this is not very important.

Reply:
I did not quite understand

Reply:
You already asked this, Nikita and I answered you in Slack.